src/torchcodec/decoders/_core/CudaDevice.cpp

```diff
@@ -225,7 +225,7 @@ void convertAVFrameToDecodedOutputOnCuda(
   auto start = std::chrono::high_resolution_clock::now();
   NppStatus status;
   if (src->colorspace == AVColorSpace::AVCOL_SPC_BT709) {
-    status = nppiNV12ToRGB_709HDTV_8u_P2C3R(
+    status = nppiNV12ToRGB_709CSC_8u_P2C3R(
```

Comment:
Drive-by: this should probably be gated by AVFrame::color_range.

https://ffmpeg.org/doxygen/trunk/pixfmt_8h.html#a3da0bf691418bc22c4bcbe6583ad589a

MPEG has limited color-range while JPEG has full color range.

We were using full for all frames before -- now we are using limited for all. Colorspace and color-range are orthogonal so probably need nested ifs here.